### ISSUE TITLE
perflib_logicaldisk metrics missing

### ISSUE THREAD

**Issue by BertHartm** [leoluk/perflib_exporter#1]
first, thanks for making this, and I realize it's alpha software.

I'm trying to get this running across a fleet of machines because I'm hitting the WMI bug that inspired this, and this seems to pretty consistently report data on machines where the WMI exporter won't.

I'm trying to reimplement my reporting on top of this as a PoC, but I don't understand why some metrics are reported and not others. For instance, I'm seeing `perflib_logicaldisk_free_megabytes`, but not the metric reported before or after that in /dump (`free_space_total` / `current_disk_queue_length` respectively)

It seems like all of Logical Disk should be enabled together, or not at all, unless I'm missing something?

Would appreciate your insight.

**Comment by leoluk** [comment]
If you can, please send me the /dump output (my mail address is in my profile). It might be an issue with how the metric names are generated from the perflib names, or perflib simply not reporting those metrics.

General note: I'm working to get this stable and production-ready, and I expect a stable version soon-ish (within 1-2 months). In the meantime, the metric names will change, so expect a breaking change or two. Glad to hear it's useful to you.

**Comment by jwenz723** [comment]
I am maybe experiencing a related issue... I am starting my exporter using the command `perflib_exporter.exe --perflib.objects=230 --log.level=debug` to simply collect metrics from the `Process` category. There are no metrics being exported, and here is the output that I am getting:

```
←[37mDEBU←[0m[0011] Skipping 230 -> %!s(uint=6) (empty merge label)  ←[37msource
←[0m="collector.go:139"
←[37mDEBU←[0m[0011] Skipping 230 -> %!s(uint=6) (empty merge label)  ←[37msource
←[0m="collector.go:139"
←[37mDEBU←[0m[0011] Skipping 230 -> %!s(uint=6) (empty merge label)  ←[37msource
←[0m="collector.go:139"
←[37mDEBU←[0m[0011] Skipping 230 -> %!s(uint=6) (empty merge label)  ←[37msource
←[0m="collector.go:139"
←[37mDEBU←[0m[0011] Skipping 230 -> %!s(uint=6) (empty merge label)  ←[37msource
←[0m="collector.go:139"
←[37mDEBU←[0m[0011] Skipping 230 -> %!s(uint=6) (empty merge label)  ←[37msource
...
```

**Comment by BertHartm** [comment]
Might be the same issue. I have 230 turned on, but I don't think I'm seeing any metrics for it

**Comment by leoluk** [comment]
I created #7 for the (separate) issue with 230.

**Comment by leoluk** [comment]
Fixed by #14
